libs/client/use-cases/dashboard/shared/ui/src/lib/navbar/navbar.component.html

```diff
@@ -1,12 +1,19 @@
-<ul *ngIf="navbar" class="navbar card-primary">
-  <li
+<nav
+  *ngIf="navbar"
```

Comment:
![navbar](https://user-images.githubusercontent.com/23154054/215512773-5fa904b5-f079-4f0e-9608-bbc78e4308d7.PNG)

I would like the right border to match the content of the navbar layout on the right. We need it to be shifted all the way to the right so that the blue border is visible